nodejs/dist/uws.js

```diff
@@ -209,9 +210,16 @@ class Server extends EventEmitter {
                 options.path = '/' + options.path;
             }
 
-            this.httpServer.on('upgrade', (request, socket, head) => {
-                if (!options.path || options.path == request.url.split('?')[0].split('#')[0]) {
-                    if (options.verifyClient) {
+            this._onServerUpgrade = (request, socket, head) => {
+
+                // check for wrong path
+                if (options.path) {
+                  var u = url.parse(request.url);
+                  if (u && u.pathname !== options.path)
+                    return;
```

Comment:
I really do not understand where node.js can stand on how to use "upgrade" request. Can you point me somewhere ? 


```
globalapp = MyAppControler.connect(new Websocket());

var liveStreamCam01 = new Websocket("/cam/hallway");//binary data only
liveStreamCam01.pipe(h264decoder).display($("#canvas1"))

var liveStreamCam02 = new Websocket("/cam/parking");//binary data only
liveStreamCam01.pipe(h264decoder).display($("#canvas2"))

globalapp.on('message', function thatHaveNothingToDoWithCameras)
```